src/meshapi/tests/test_nn.py

```diff
@@ -429,25 +429,26 @@ def test_nn_valid_low_install_number_used_nn(self):
             content_type="application/json",
         )
 
-        code = 201
+        code = 409
         self.assertEqual(
             code,
             response.status_code,
             f"status code incorrect for test_nn_valid_install_number. Should be {code}, but got {response.status_code}",
         )
 
-        resp_nn = json.loads(response.content.decode("utf-8"))["network_number"]
-        expected_nn = 101
         self.assertEqual(
-            expected_nn,
-            resp_nn,
-            f"nn incorrect for test_nn_valid_install_number. Should be {expected_nn}, but got {resp_nn}",
+            json.loads(response.content.decode("utf-8")),
+            {
+                "detail": "Invalid install status for NN Assignment. "
+                "Re-submit the join form to create a new install number"
```

Comment:
Just a thought: Should we have some one-click way to re-submit a join form for an install whose NN has been re-assigned?

Reply:
Seems like such an edge case, copy pasting from another browser tab is really not that bad